importers/src/BeImportFileCreator.class.ts

```diff
@@ -115,6 +119,10 @@ export class BeImportFileCreator {
             // write index file
             writeFileSync(`${targetDir}/index.json`, JSON.stringify(versionIndex));
 
+            if (options && options.skipCompression) {
+                return;
```

Comment:
i guess you should add a `pResolve(` with maybe a list of created files(-paths)? in any case, like this the returned promise does not resolve (as far as i can see)

Reply:
Updated to explicitly resolve the promise 👍 

Didn't add any parameters to the resolve function, because the consumer doesn't need to know the list of created files (I can add it later if it does end up needing it)